telemetry/definitions/commonDefinitions.json

```diff
@@ -3816,14 +3860,14 @@
         {
             "name": "codeTransform_logApiError",
             "passive": true,
-            "description": "Deprecated - avoid logging every service call",
+            "description": "An API-side error has occurred.",
             "metadata": [
                 {
-                    "type": "codeTransformApiErrorMessage",
+                    "type": "codeTransformApiNames",
```

Comment:
why is ErrorMessage changed to APi names instead of `codeTransformApiErrorMessage`

Reply:
This has been addressed in a more recent commit.